cypress.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from 'cypress';
+import * as dotenv from 'dotenv';
+
+dotenv.config({ path: '.env' });
+dotenv.config();
+
+export default defineConfig({
+  fixturesFolder: 'src/test/cypress/fixtures',
+  env: {
+    // stripes username and password
+    diku_login: process.env.DIKU_USERNAME,
+    diku_password: process.env.DIKU_PASSWORD,
+    okapi_tenant: 'diku',
+    calendar_api_url: 'https://folio-dev-bama-okapi.ci.folio.org/calendar/calendars'
```

Comment:
Please use the snapshot environment; `folio-dev-bama` won't be around forever
```suggestion
    calendar_api_url: 'https://folio-snapshot-okapi.dev.folio.org/calendar/calendars'
```